pkg/mdformatter/linktransformer/link.go

```diff
@@ -175,7 +178,30 @@ func NewValidator(logger log.Logger, linksValidateConfig []byte, anchorDir strin
 	v.c.OnError(func(response *colly.Response, err error) {
 		v.rMu.Lock()
 		defer v.rMu.Unlock()
-		v.remoteLinks[response.Ctx.Get(originalURLKey)] = errors.Wrapf(err, "%q not accessible; status code %v", response.Request.URL.String(), response.StatusCode)
+		retriesStr := response.Ctx.Get(retryKey)
+		retries, _ := strconv.Atoi(retriesStr)
+		switch response.StatusCode {
+		case 429:
+			if retries <= 2 {
+				response.Ctx.Put(retryKey, strconv.Itoa(retries+1))
+				retryAfter, convErr := strconv.Atoi(response.Headers.Get("Retry-After"))
+				if convErr == nil {
+					time.Sleep(time.Duration(retryAfter) * time.Second)
+				}
+
+				retryErr := response.Request.Retry()
```

Comment:
I might think we need default back of maybe?

Reply:
well default if no retry-after is set? maybe something like 1s

Reply:
Oh okay! Will try to implement this!